tests/layer_tests/pytorch_tests/test_bucketize.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import torch
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestBucketize(PytorchLayerTest):
+
+    def _prepare_input(self, input_shape, boundaries_range, input_dtype, boundaries_dtype, out_int32):
+        if out_int32:
+            output_dtype = "int32"
+        else:
+            output_dtype = "int64"
+        return (
+            np.random.randn(*input_shape).astype(input_dtype), 
+            np.arange(*boundaries_range).astype(boundaries_dtype),
+            np.zeros(input_shape).astype(output_dtype))
+
+    def create_model(self, out_int32, right, is_out):
+        class aten_bucketize(torch.nn.Module):
+
+            def __init__(self, out_int32, right, is_out) -> None:
+                super().__init__()
+                self.out_int32 = out_int32
+                self.right = right 
+                self.is_out = is_out
+
+            def forward(self, input, boundaries, output):
+                if self.is_out:
+                    return torch.bucketize(input, boundaries, out_int32=self.out_int32, right=self.right, out=output)
+                else:
+                    return torch.bucketize(input, boundaries, out_int32=self.out_int32, right=self.right)
+
+        ref_net = None
+
+        return aten_bucketize(out_int32, right, is_out), ref_net, "aten::bucketize"
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    @pytest.mark.precommit_ts_backend
+    @pytest.mark.precommit_fx_backend
```

Comment:
```suggestion
```

You didn't add bucketize to fx ops, so it may be unsupported, but test will pass. Lets not add this marks without reason.